exercises/triangle/description.md

```diff
@@ -2,6 +2,16 @@ The program should raise an error if the triangle cannot exist.
 
 ## Hint
 
-The sum of the lengths of any two sides of a triangle always exceeds or 
-is equal to the length of the third side, a principle known as the _triangle
+In mathematics, a degenerate case is a limiting case in which an element of a
+class of objects is qualitatively different from the rest of the class and hence
+belongs to another, usually simpler, class.
+
+1. If the sum of the lengths of any two sides of a triangle is equal to the
+   length of the third side, there exists a degenerate triangle.
+
+2. The sum of the lengths of any two sides of a triangle always exceeds or is
```

Comment:
It's interesting because it leaves it open to the student what to do with the degenerate triangles. You gave a possible option earlier in our discussion: It shouldn't be possible to create them because they are different. Others may choose to allow them, and classify them as either scalene or isosceles (they can't be equilateral unless all sides are zero). Others may make degenerate a separate case altogether. We like to see variety, so leaving it open seems like a good idea, which is another reason I said :+1: